.github/workflows/docker-build-lint-test.yml

```diff
@@ -12,16 +11,18 @@ jobs:
     runs-on: ubuntu-latest
     env:
       IMAGE_TAG: ${{ github.sha }}
+      OORB_DATA: /tmp/oorb/data
+  
     steps:
       - name: Checkout git repo
         uses: actions/checkout@v4
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+        uses: docker/setup-buildx-action@v3
         with:
           install: true
       - name: Build
         run: docker build --load -t thor:$IMAGE_TAG .
       - name: Lint
-        run: docker run -i thor:$IMAGE_TAG pre-commit run --all-files
+        run: docker run -i thor:$IMAGE_TAG pdm lint
```

Comment:
Maybe we can dump the docker builds altogether with the simpler pyoorb / numpy install?